cluster-up/cluster/kind-1.30/provider.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
This seems not related to the PR

Reply:
good catch, some other local change got mixed into this PR, will remove